development/docker-compose.dev.yml

```diff
@@ -20,6 +20,10 @@ services:
       # - "../../nautobot/nautobot:/usr/local/lib/python3.11/site-packages/nautobot"
 
     healthcheck:
+      interval: "30s"
```

Comment:
I purposely removed this, as I hate it on the console all the time

Reply:
cc.: @whitej6

Reply:
I’m not going to lose sleep on this, but my preference would be to remove from upstream, I don’t know why we would want this on dev

Reply:
Let's ask in dev standards. I personally agree with Ken